_posts/2021-03-09-stellar-issue-asset.md

```diff
@@ -0,0 +1,93 @@
+---
+layout: post
+title: How to issue an asset on Stellar
+subtitle: AFFEN goes live
+tags: [poc, blockchain, stellar, token, asset, github]
+author: gmzabos
+---
+
+## Intro - what is Stellar?
+
+[Stellar](http://stellar.org/){:target="_blank"} is an open-source network for currencies and payments. It is public, there is no "real" owner as it is a decentralized network. See the [Intro](https://stellar.org/learn/intro-to-stellar){:target="_blank"} on their website, as it speaks for itself better than i could do.
+
+## Intro - why do we issue our own asset?
+
+We came up with the idea while experimenting with Stellar (and other blockchains) and decided to create an asset (or "token") to hook it up to [codeaffen](https://github.com/codeaffen){:target="_blank"} on [github](https://github.com/){:target="_blank"}. The token would function within that ecosystem as a reward for the sake of gamification while submitting code & contributing in other ways to the project. Every good role-playing game has its own XP, you get the idea...
```

Comment:
I would only place one link into the text like `[codeaffen on github](https://github.com/codeaffen)...` We want visitors to go to our repo overview not to global github.

Reply:
OK